src/sentry/api/base.py

```diff
@@ -234,12 +249,6 @@ def dispatch(self, request, *args, **kwargs):
                     # setup default access
                     request.access = access.from_request(request)
 
-            if (
-                getattr(self.request.successful_authenticator, "token_name", None)
-                == TokenAuthentication.token_name
-            ):
-                request._metric_tags["backend_request"] = True
-
```

Comment:
I see that you added this earlier, are we just no longer using it?

Reply:
From looking at datadog the last week, it more looks like this didn't work. We're now doing the same thing with logging, so I don't see a point of keeping this around.